enclave-runner/src/usercalls/mod.rs

```diff
@@ -776,14 +776,15 @@ impl EnclaveState {
         };
 
         let select_fut =
-            futures::future::select(return_future.boxed(), io_future.boxed()).map( |either| {
+            futures::future::select(return_future.boxed_local(), io_future.boxed_local()).map( |either| {
```

Comment:
Please add a comment that you know `io_future` isn't going to resolve, but that's intentional.